Dockerfile-slim

```diff
@@ -0,0 +1,52 @@
+# Stage 1: Build
+FROM rust:latest AS builder
+
+# Install build dependencies
+RUN apt-get update && apt-get install -y \
+    libpcap-dev \
+    iproute2 \
+    # linux-headers \
+    && rustup toolchain install stable \
+    && rustup toolchain install nightly --component rust-src \
+    && cargo install bpf-linker \
+    && rm -rf /var/lib/apt/lists/*
+
+# Set environment variables
+ENV PATH="/root/.cargo/bin:${PATH}"
+ENV RUST_LOG=info
```

Comment:
This environment variable is already added below in the running phase, can be deleted here.